### PR TITLE
Perf/UI

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Update FE structure

### CODE REVIEW

File: webapp/src/app/[locale]/credential/page.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+interface RequestResolvedProps {
+  session_id: string
+}
+
+const RequestResolvedComponent = ({ session_id }: RequestResolvedProps) => {
+  const locale = useLocale()
+  const t = useTranslations('IndexPage')
+  const [snackbarOpen, setSnackbarOpen] = React.useState(false)
+  const [loadState, { loading, data }] = useLazyQuery(lazyQuery, {
```
[the remark this answers is not in the document]
Comment (leisterfrancisco) [reply webapp/src/app/[locale]/credential/page.tsx]: already fixed

File: webapp/src/app/[locale]/components/TableCredential/index.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+import TableHead from '@mui/material/TableHead'
+import TablePagination from '@mui/material/TablePagination'
+import TableRow from '@mui/material/TableRow'
+import TableSortLabel from '@mui/material/TableSortLabel'
+import Toolbar from '@mui/material/Toolbar'
+import Typography from '@mui/material/Typography'
+import Paper from '@mui/material/Paper'
+import IconButton from '@mui/material/IconButton'
+import Tooltip from '@mui/material/Tooltip'
+import SearchIcon from '@mui/icons-material/Search'
```
[the remark this answers is not in the document]
Comment (leisterfrancisco) [reply webapp/src/app/[locale]/components/TableCredential/index.tsx]: already fixed

File: webapp/src/app/[locale]/components/TableCredential/index.tsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+import { visuallyHidden } from '@mui/utils'
```
[the remark this answers is not in the document]
Comment (leisterfrancisco) [reply webapp/src/app/[locale]/components/TableCredential/index.tsx]: already fixed

File: webapp/src/app/[locale]/components/TableCredential/index.tsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TablePagination from '@mui/material/TablePagination'
+import TableRow from '@mui/material/TableRow'
+import TableSortLabel from '@mui/material/TableSortLabel'
+import Toolbar from '@mui/material/Toolbar'
+import Typography from '@mui/material/Typography'
+import Paper from '@mui/material/Paper'
+import IconButton from '@mui/material/IconButton'
+import Tooltip from '@mui/material/Tooltip'
```
[the remark this answers is not in the document]
Comment (leisterfrancisco) [reply webapp/src/app/[locale]/components/TableCredential/index.tsx]: already fixed

File: webapp/src/app/[locale]/components/TableCredential/index.tsx [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TablePagination from '@mui/material/TablePagination'
+import TableRow from '@mui/material/TableRow'
+import TableSortLabel from '@mui/material/TableSortLabel'
+import Toolbar from '@mui/material/Toolbar'
+import Typography from '@mui/material/Typography'
+import Paper from '@mui/material/Paper'
+import IconButton from '@mui/material/IconButton'
```
[the remark this answers is not in the document]
Comment (leisterfrancisco) [reply webapp/src/app/[locale]/components/TableCredential/index.tsx]: already fixed

File: webapp/src/app/[locale]/components/TableCredential/index.tsx [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+import Box from '@mui/material/Box'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TablePagination from '@mui/material/TablePagination'
+import TableRow from '@mui/material/TableRow'
+import TableSortLabel from '@mui/material/TableSortLabel'
+import Toolbar from '@mui/material/Toolbar'
```
[the remark this answers is not in the document]
Comment (leisterfrancisco) [reply webapp/src/app/[locale]/components/TableCredential/index.tsx]: already fixed

File: webapp/src/app/[locale]/components/TableCredential/index.tsx [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+import { useLocale, useTranslations } from 'next-intl'
+import Box from '@mui/material/Box'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TablePagination from '@mui/material/TablePagination'
+import TableRow from '@mui/material/TableRow'
+import TableSortLabel from '@mui/material/TableSortLabel'
```
[the remark this answers is not in the document]
Comment (leisterfrancisco) [reply webapp/src/app/[locale]/components/TableCredential/index.tsx]: already fixed

**Review by kuronosec — APPROVED** [review]
LGTM

**Pull request merged**